src/lib/Components/Consignments/Screens/Welcome.tsx

```diff
@@ -1,114 +1,153 @@
 import React from "react"
-import { Dimensions, Image, NavigatorIOS, Route, ScrollView, ViewProperties } from "react-native"
+import { Image, LayoutChangeEvent, NavigatorIOS, Route, ScrollView, TextProperties, ViewProperties } from "react-native"
 
 import { Schema, screenTrack } from "lib/utils/track"
 import CloseButton from "../Components/CloseButton"
 import ConsignmentBG from "../Components/ConsignmentBG"
 import { Button } from "../Components/FormElements"
-import { LargeHeadline } from "../Typography"
+import * as Typography from "../Typography"
 import Overview from "./Overview"
 
 import { Fonts } from "lib/data/fonts"
-import styled from "styled-components/native"
+import styled, { StyledFunction } from "styled-components/native"
+
+const View: StyledFunction<DeviceProps & ViewProperties> = styled.View
+const Text: StyledFunction<DeviceProps & TextProperties> = styled.Text
+
+interface DeviceProps {
+  isTiny: boolean
+  isPad: boolean
+  isPortrait: boolean
+}
 
 interface Props extends ViewProperties {
   navigator: NavigatorIOS
   route: Route
 }
 
-const isPad = Dimensions.get("window").width > 700
+interface State {
+  deviceProps?: DeviceProps
+}
 
-const VerticalLayout = styled.View`
+const VerticalLayout = View`
   flex: 1;
   flex-direction: column;
-  align-items: ${isPad ? "center" : "stretch"};
+  align-items: ${({ isPad }) => (isPad ? "center" : "stretch")};
 `
-const Listpad = styled.View`
-  align-items: center;
-  padding-top: 100;
-`
-const Listphone = styled.View`
-  margin-bottom: 100;
-  margin-left: 20;
-  margin-top: 50;
-  margin-right: 50;
+
+const List = View`
+  align-self: center;
+  width: ${({ isTiny }) => (isTiny ? 280 : 330)};
+  margin-top: 0;
+  margin-bottom: 0;
 `
 
-const HorizontalLayout = styled.View`
+const HorizontalLayout = View`
   flex-direction: row;
-  margin-left: ${isPad ? 0 : 20};
-  margin-top: ${isPad ? 40 : 10};
-  margin-bottom: ${isPad ? 0 : 10};
+  margin-left: 0;
```

Comment:
The left margin was there because the icons felt too unbalanced / heavy on the left, so Katarina had asked me to center them a bit more. (e.g. as per the iPhone 6 screenshot). Is this being changed in the new design?

Reply:
All phones have 20pt margins on the left side, except it’s being set on the list rather than the row. I’m not sure I follow completely because you only mention the iPhone 6, do the 4s and 5s not look right?

Reply:
Oops, the list has no margin, it has a fixed width (aligned with the title which has hardcoded letter-spacing values), which effectively add up to 20pt margin.

Reply:
I meant that it's more prominent in the iPhone 6, because as the screen grows, there's more space available which sits on the right :o

Reply:
nevermind maybe I'm just imagining it. so many pixels lol.

Reply:
I know, right? My head is spinning